game/songs.cc

```diff
@@ -97,9 +97,13 @@ void Songs::LoadCache() {
     	return;
     }
 
+    auto stringList = config["paths/songs"].sl();
+
     for(auto const& song : jsonRoot.as_array()) {
     	struct stat buffer;
-    	if(stat(song.at("TxtFile").as_string().c_str(), &buffer) == 0) {
+    	auto songPath = song.at("TxtFile").as_string();
+    	auto isSongPathInConfiguredPaths = std::find_if(stringList.begin(), stringList.end(), [songPath](const std::string& stringListItem) { return songPath.find(stringListItem) != std::string::npos; }) != stringList.end();
```

Comment:
please use some enters here, it's pretty hard to read when you cram this amount of stuff in a single line.

Reply:
Fixed